hw/top_earlgrey/dv/chip_dif_tests.hjson

```diff
@@ -56,18 +56,6 @@
       sw_images: ["sw/device/tests/kmac_smoketest:1"]
       en_run_modes: ["sw_test_mode"]
     }
-    {
```

Comment:
Why are these removed instead of being renamed?

Reply:
PTAL at #7976, specifically the portion that says "There will be one KMAC smoke test..."  and the "Organizing Chip-Level Test Code" design doc that was discussed and approved at the Si WG two weeks ago.

Reply:
Right, but the tests exist - they are just renamed. They need to be added back to the adjoining `chip_sim_cfg.hjson`.

Reply:
Added this a task under "Summary of Desired Changes to sw/device/tests" in #7976, and will complete in a future PR to close out #7976.